app/src/main/AndroidManifest.xml

```diff
@@ -25,4 +26,6 @@
         </activity>
     </application>
 
+    <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
Permissions are declared at the top of the manifest.

Reply:
fixed